File: update-gif.js
const fs = require("fs");
const path = require("path");

const gifFolder = "./gif";
const baseUrl = "https://github.com/miguel-lamazares/miguel-lamazares/blob/main/gif/";

const gifs = fs.readdirSync(gifFolder).filter(file => file.endsWith(".gif"));

if (gifs.length === 0) {
  console.error("Nenhum gif encontrado na pasta gif/");
  process.exit(1);
}

const escolhido = gifs[Math.floor(Math.random() * gifs.length)];
const gifUrl = `${baseUrl}${encodeURIComponent(escolhido)}`;

let readme = fs.readFileSync("README.md", "utf8");

readme = readme.replace(
  /<!-- start-cs2 -->[\s\S]*?<!-- end-cs2 -->/,
  `<!-- start-cs2 -->\n<img height="150" src="${gifUrl}"/>\n<!-- end-cs2 -->`
);

fs.writeFileSync("README.md", readme);
console.log(`GIF atualizado para: ${escolhido}`);

File: update_gif.py
import os
import random

# Caminho para a pasta com os GIFs
gif_folder = 'gif'
gif_files = [f for f in os.listdir(gif_folder) if f.endswith('.gif')]

# Escolher um GIF aleatório
random_gif = random.choice(gif_files)

# Caminho para o README.md
readme_path = 'README.md'

# Substituir a imagem no README
with open(readme_path, 'r') as file:
    content = file.read()

# Substituir o link do GIF no README
new_content = content.replace("gif/1.gif", f"gif/{random_gif}")

# Salvar a alteração no README
with open(readme_path, 'w') as file:
    file.write(new_content)

print(f"Atualizado para o GIF {random_gif}")
